app/views/admins/show_bills.html.erb

```diff
@@ -0,0 +1,12 @@
+All Bills | <%= link_to "Back To Admin Page", admins_path() %>
```

Comment:
You don't need to add empty parentheses to your `index` path.

```suggestion
All Bills | <%= link_to "Back To Admin Page", admins_path %>
```

Reply:
removed ()